torchvision/models/detection/ssd.py

```diff
@@ -197,9 +198,9 @@ def __init__(self, backbone: nn.Module, anchor_generator: DefaultBoxGenerator,
         self.proposal_matcher = det_utils.SSDMatcher(iou_thresh)
 
         if image_mean is None:
-            image_mean = [0.485, 0.456, 0.406]
+            image_mean = (0.485, 0.456, 0.406)
```

Comment:
Why did you change the type? Shouldn't `List[float]` also be ok?

Reply:
Hmmm. I believe it's because he defines the type as `Tuple[float, ...]` instead of `List[float]`. I think over the library we use a lot of different ways for this, perhaps it would be best to keep one and stick with it. I think `Tuple[float, ...]` is only used in a handful of methods at `functional_pil.py` while `List[float]` is more common.

Reply:
Hmm, `List[float]` too would have worked fine. I will try changing it once, I don't recall the error. 

Anyway about tuple.
Since we are not modifying `image_mean` or `image_std`, it can be good to leave as tuple. As such tuple are immutable so a good standard choice. Tuple can save a bit of memory (too little but yeah).

Reply:
The best choice would be `Sequence[float]` since we do not care about the actual type. But that is not supported by torchscript. Thus, I would simply go with whatever we had or in case you add something new go with what we use the most throughout the code base.

Reply:
Let's make a convention of using `List[float]`. I'm editing the code to keep consistency with List[float]